bin/fusioninventory-injector

```diff
@@ -125,6 +131,9 @@ sub sendContent {
         print $res->is_success() ?
             "OK\n" : "ERROR: " . $res->status_line() . "\n";
     }
+    unless ($res->is_success()) {
+        die "Content has not been sent. HTTP status is : " . $res->status_line;
```

Comment:
It's better to handle a counter and return an error if injected count is not the total count.